pkg/config/config_test.go

```diff
@@ -1049,3 +1049,83 @@ func (s *MySuite) TestValidatorConfigCheck(c *C) {
 		c.Check(err, ErrorMatches, "only 3 inputs \\[in0 in1 in2\\] should be provided to test_project_exists")
 	}
 }
+
+func (s *MySuite) TestCheckBackends(c *C) {
+	// Helper to create blueprient with backend blocks only (first one is defaults)
+	// and run checkBackends.
+	check := func(d TerraformBackend, gb ...TerraformBackend) error {
+		gs := []DeploymentGroup{}
+		for _, b := range gb {
+			gs = append(gs, DeploymentGroup{TerraformBackend: b})
+		}
+		bp := Blueprint{
+			TerraformBackendDefaults: d,
+			DeploymentGroups:         gs,
+		}
+		return checkBackends(bp)
+	}
+	dummy := TerraformBackend{}
+
+	{ // Absent
+		c.Check(checkBackends(Blueprint{}), IsNil)
+	}
+
+	{ // Dummies
+		c.Check(check(dummy, dummy, dummy), IsNil)
+	}
+
+	{ // OK
+		b := TerraformBackend{
+			Type: "gcs",
+			Configuration: map[string]interface{}{
+				"bucket":                      "trenta",
```

Comment:
nit: use something more like foo/bar or "not-a-bucket" etc.

Reply:
I assume your intentions is to "reduce mental load". I see how it can be applicable for this particular test, but readability gains are too minuscule, I would argue not existent.
On another hand overuse  of "foo/bar/buzz" can hinder both readability and debugability of a test.
So I'm trying to avoid using "foo/bar/buzz" in code and use diverse random relatively short words instead.
If I misunderstood your comment intention, please correct me.